pkg/ee/metering/reconcile.go

```diff
@@ -136,6 +137,10 @@ func reconcileMeteringReportConfigurations(ctx context.Context, client ctrlrunti
 		return err
 	}
 	if err := mc.SetBucketLifecycle(ctx, bucket, config); err != nil {
+		// Ignore conflict error in case lock after previous reconciliation process still exists.
```

Comment:
Please see if you can cast the error to some concrete type and check its properties. A string comparision on errors is too brittle and will break soon. It looks like the Minio client has some HttpError stuff: https://github.com/minio/minio-go/search?q=conflict